cmd/gazelle/update-repos.go

```diff
@@ -175,7 +175,16 @@ func updateRepos(args []string) (err error) {
 	var newGen []*rule.Rule
 	genForFiles := make(map[*rule.File][]*rule.Rule)
 	emptyForFiles := make(map[*rule.File][]*rule.Rule)
+	ruleSet := make(map[string]*rule.Rule)
 	for _, r := range gen {
+		if existingRule, exist := ruleSet[r.Name()]; exist {
```

Comment:
Nit: rename `exist` to `ok`. It's a bit more idiomatic. Alternatively, just check whether `existingRule` is nil.